datahub-web-react/src/app/settingsV2/features/Features.tsx

```diff
@@ -99,14 +89,8 @@ export const Features = () => {
         <Page>
             <SourceContainer>
                 <Container>
-                    <div>
-                        <Title level={2}>Features</Title>
-                        <Typography.Paragraph type="secondary">
-                            Explore and configure specific features
-                        </Typography.Paragraph>
-                    </div>
+                    <PageTitle title="Features" subTitle="Explore and configure specific features" />
```

Comment:
Note: we need to start being consistent about whether we put a punctuation at the end of these sub-headers